vite-project/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,15 @@
+import "../style/SearchBar.css";
+
+function SearchBar() {
+  return (
+    <div className="searchbar-section">
+      <input
+        className="searchbar-input"
+        type="search"
+        placeholder="링크를 검색해 보세요."
+      ></input>
```

Comment:
input과 같이 children이 없는 경우에는 self-closing tag로 작성하는게 좋겠습니다.
```suggestion
      />
```